src/assets/v1.3.0/gnosis_safe_l2.json

```diff
@@ -105,6 +105,7 @@
     "9728": "0xfb1bffC9d739B8D520DaF37dF666da4C687191EA",
     "10000": "0xfb1bffC9d739B8D520DaF37dF666da4C687191EA",
     "10001": "0xfb1bffC9d739B8D520DaF37dF666da4C687191EA",
+    "10081": "0xB19D6FFc2182150F8Eb585b79D4ABcd7C5640A9d",
```

Comment:
This doesn't seem to match the expect address

Reply:
Hi @mmv08 Thank you for your reviewing. I've already update it